lib/commands/clipboard.js

```diff
@@ -0,0 +1,21 @@
+import log from '../logger';
+import { util } from 'appium-support';
+
+
+let extensions = {}, commands = {};
+
+commands.mobileSetClipboard = async function (opts = {}) {
+  if (!util.hasValue(opts.content)) {
+    log.errorAndThrow(`The 'content' argument is mandatory`);
+  }
+  return await this.uiautomator2.jwproxy.command('/appium/clipboard/set', 'POST', opts);
+};
+
+commands.mobileGetClipboard = async function (opts = {}) {
+  return await this.uiautomator2.jwproxy.command('/appium/clipboard/get', 'POST', opts);
```

Comment:
How the `/get` endpoint method uses `POST` and not `GET`?

Reply:
even so, could we include that as a url param like `/appium/clipboard/get/:contentType`?

Reply:
I find POST more flexible %)

Reply:
Can it not use query strings? (e.g.: `/appium/clipboard?contentType=<...>&...`